src/js/components/Video/stories/typescript/TheaterMode.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+
+import { Box, Video } from 'grommet';
+import { Monitor } from 'grommet-icons/icons/Monitor';
+
+const TheaterMode = (props) => {
+  const [theaterMode, setTheaterMode] = useState(false);
+  const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
```

Comment:
This looks like a hack. There should be a way to get the desired behavior without having to resort to this approach. You might want to remove the `fill="vertical"` on the Box, and potentially the `flex={true}`.